app/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/RecordsRepository.scala

```diff
@@ -143,6 +150,21 @@ class RecordsRepository @Inject() (
     searchTerm match {
       case Some(value) =>
         field match {
+
+          case Some("traderRef") =>
```

Comment:
do you need to update the traderRef in byComCodeOrGoodsDescriptionOrTraderRef aswell?

Reply:
That's only used by the search and already seems to work with case insensitive - I guess because it uses regex rather than exact match. So think we are fine for now :)